### PR TITLE
1411 visualization data leak

### PR DESCRIPTION
**Pull request by mlissner**

### CODE REVIEW

File: cl/visualizations/views.py [hunk 1]
```diff
@@ -32,10 +32,14 @@ def render_visualization_page(request, pk, embed):
     status = None
     if viz.deleted:
         status = statuses.HTTP_410_GONE
+        title = u"Visualization deleted by owner"
```
Comment (flooie) [inline cl/visualizations/views.py:35]: Owner feels strange for some reason.  Creator?

File: cl/visualizations/views.py [hunk 2]
```diff
@@ -32,10 +32,14 @@ def render_visualization_page(request, pk, embed):
...
     else:
         if viz.published is False and viz.user != request.user:
             # Not deleted, private and not the owner
             status = statuses.HTTP_401_UNAUTHORIZED
+            title = u"Private visualization"
```
Comment (flooie) [inline cl/visualizations/views.py:40]: Not sure this matters, but do we want to let people know that it exists in the title?

Comment (mlissner) [reply cl/visualizations/views.py:40]: Should be OK. They only have a URL with an ID in it. The most you can learn is how many visualizations have been created. Doesn't say by whom or anything like that.

File: cl/visualizations/views.py (hunk 1 above)
Comment (mlissner) [reply cl/visualizations/views.py]: Sure. Done.

**Review by flooie — COMMENTED** [review]
Looks good to me.

**Pull request merged**